api_app/api/dependencies/database.py

```diff
@@ -40,8 +40,8 @@ async def get_store_key(credential) -> str:
         async with CosmosDBManagementClient(
             credential,
             subscription_id=config.SUBSCRIPTION_ID,
-            base_url=cloud.get_resource_manager_endpoint(),
-            credential_scopes=cloud.get_resource_manager_credential_scopes()
+            base_url=config.RESOURCE_MANAGER_ENDPOINT,
+            credential_scopes=[config.RESOURCE_MANAGER_ENDPOINT + ".default"]
```

Comment:
What do you think about making this into an f-style string manipulation? 
We could also consider moving it to its own computed config entry in the config.py file and then it will be appearing just once.